frontend/src/pages/RunList.test.tsx

```diff
@@ -453,7 +464,9 @@ describe('RunList', () => {
 
   it('shows pipeline version name', async () => {
     mockNRuns(1, {
-      pipeline_version_id: 'testversion1',
+      pipeline_version_reference: {
+        pipeline_version_id: 'testversion1',
```

Comment:
Whenever `pipeline_version_id` present, `pipeline_id` should present as well to make `pipeline_version_reference` a valid object, even if your test might not need to check the other part.

Reply:
Agree with your point; however, (Correct me if I am wrong), I think making both changes (adding `pipeline_id` in the implementation and test) in pipeline API integration rather separating into two PRs is better to manage the code changes.

Reply:
This seems to be a unit test that you're providing a test input, if you just add one line of `pipeline_id` here, would it require any other change? I might be wrong, but I guess it won't break anything, and you have a valid test input that you don't need to change again soon after.